packages/babel-preset-react/src/index.js

```diff
@@ -8,16 +8,22 @@ import transformReactJSXSelf from "@babel/plugin-transform-react-jsx-self";
 export default declare((api, opts) => {
   api.assertVersion(7);
 
+  let { pragma, pragmaFrag } = opts;
+
   const {
-    pragma,
-    pragmaFrag,
     pure,
     throwIfNamespace = true,
     useSpread,
     runtime = "classic",
     importSource,
   } = opts;
 
+  // TODO: (Babel 8) Remove setting these defaults
+  if (runtime === "classic") {
+    pragma = pragma ?? "React.createElement";
+    pragmaFrag = pragmaFrag ?? "React.Fragment";
```

Comment:
Since we are trying to restore the old behavior, we should probably use `||` again.

Reply:
:+1: